app/src/main/java/com/example/smartplugconfig/NetworkRepository.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.smartplugconfig
+
+import androidx.compose.runtime.mutableStateListOf
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+object NetworkRepository{
+
+    private val _mifiNetworks = MutableStateFlow<List<String>>(emptyList())
+    val mifiNetworks: StateFlow<List<String>> get() = _mifiNetworks
```

Comment:
Regarding the setupMqttBroker(context: Context)
We need the logic to be docuwmented to explain what is "packet.topicName == "stat/smartPlug/STATUS8" and packet.topicName == "tele/smartPlug/SENSOR". We need a digram with the flow of miwifi and plugwifi and mqtt communication to be draw. Based on this will be much easier to relocate the logic into the connected app

Reply:
I will add a few comments and make sure this is included in the handover page on confluence